build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         maven { url "http://dl.bintray.com/melix/thirdparty-apache" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:2.3.0'
```

Comment:
This does seem to be the case. Points still stands, but 2.3.1 has some bugfixes. Will update.
Also i'm an idiot.